UIforETW/Utility.cpp

```diff
@@ -40,26 +166,35 @@ std::vector<std::wstring> split(const std::wstring& s, char c)
 	return result;
 }
 
-std::vector<std::wstring> GetFileList(const std::wstring& pattern, bool fullPaths)
+std::vector<std::wstring> GetFileList(const std::wstring& pattern, const bool fullPaths)
 {
-	std::wstring directory;
-	if (fullPaths)
-		directory = GetDirPart(pattern);
+	const std::wstring directory = (fullPaths ? GetDirPart( pattern ) : L"");
+
+	//may not pass an empty string to FindFirstFileEx
+	UIETWASSERT(pattern.length() > 0);
+
+	//string passed to FindFirstFileEx must not end in a backslash

```

Comment:
Comments should have a space and then a capital letter. 'cause that's what the other comments have.